release/cloud_builder.sh

```diff
@@ -142,6 +145,25 @@ if [ "${BUILD_DOCKER}" == "true" ]; then
   ./bin/push-docker           -h " " -t "${TAG_NAME}" -b -o "${OUTPUT_PATH}"
 fi
 if [ "${BUILD_DEBIAN}" == "true" ]; then
-  ./bin/push-debian.sh -c opt -v "${TAG_NAME}" -o "${OUTPUT_PATH}"
+  ./bin/push-debian.sh -c opt -v "${TAG_NAME}" -o "${OUTPUT_PATH}/deb"
 fi
 popd
+
+# remove the -release suffix from the basename in debian images
+if [ "${BUILD_DEBIAN}" == "true" ]; then
```

Comment:
Instead of doing that, can we just get rename of the *-release pkg_deb ? Those are creating confusion. I think there was an issue to combine those.

Reply:
There is #1118 and it's quite important - to avoid manual copying